src/libs/actions/PersonalDetails.js

```diff
@@ -183,6 +178,19 @@ function getFromReportParticipants(reports) {
 // When the app reconnects from being offline, fetch all of the personal details
 NetworkConnection.onReconnect(fetch);
 
+// Refresh the personal details and timezone now and then every 30 minutes because there is no
+// pusher event that sends updated personal details data yet
+// See https://github.com/Expensify/ReactNativeChat/issues/468
+fetch();
```

Comment:
FYI we are already calling this in `HomePage` but it was imported with a different name.

https://github.com/Expensify/Expensify.cash/blob/7823b07422f3bbc935f9e9f57f8e60e099c12a11/src/pages/home/HomePage.js#L28
https://github.com/Expensify/Expensify.cash/blob/7823b07422f3bbc935f9e9f57f8e60e099c12a11/src/pages/home/HomePage.js#L86-L87

This is one of the reasons I don't like renaming imports. Makes it almost impossible to figure this kind of thing out.

Reply:
Should I just remove it from there? Isn't it better to fetch them here?

Reply:
🤷‍♂️  Not sure. I think it makes the most sense to fetch them in one spot and also where a bunch of other "initialize-y" code is running i.e. `HomePage`. I wouldn't expect an action file to call its own methods since they are mostly exported and used in components and other places..